python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 2162
   - name: g_decl
     "@id": "#Object.g_decl"
     datatype: double
     description: Position in Declination. Measured on g-band.
     fits:tunit: deg
     ivoa:ucd: pos.eq.dec
+    tap:principal: 0
+    tap:column_index: 2002
   - name: g_extendedness
     "@id": "#Object.g_extendedness"
     datatype: double
     description: Set to 1 for extended sources, 0 for point sources. Measured on g-band.
     fits:tunit:
+    tap:principal: 1
+    tap:column_index: 2163
   - name: g_extendedness_flag
     "@id": "#Object.g_extendedness_flag"
     datatype: boolean
     description: Set to 1 for any fatal failure. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2164
   - name: g_free_cModelFlux
     "@id": "#Object.g_free_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2165
   - name: g_free_cModelFluxErr
     "@id": "#Object.g_free_cModelFluxErr"
     datatype: double
     description: Flux uncertainty from the final cmodel fit. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2166
   - name: g_free_cModelFlux_flag
     "@id": "#Object.g_free_cModelFlux_flag"
     datatype: boolean
     description: Flag set if the final cmodel fit (or any previous fit) failed. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2167
   - name: g_free_cModelFlux_inner
     "@id": "#Object.g_free_cModelFlux_inner"
     datatype: double
     description: Flux within the fit region, with no extrapolation. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2168
   - name: g_free_psfFlux
     "@id": "#Object.g_free_psfFlux"
     datatype: double
     description: Flux derived from linear least-squares fit of PSF model. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2169
   - name: g_free_psfFluxErr
     "@id": "#Object.g_free_psfFluxErr"
     datatype: double
     description: Flux uncertainty derived from linear least-squares fit of PSF model. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2170
   - name: g_free_psfFlux_flag
     "@id": "#Object.g_free_psfFlux_flag"
     datatype: boolean
     description: General Failure Flag. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2171
   - name: g_fwhm
     "@id": "#Object.g_fwhm"
     datatype: double
     description: HSM moments. Measured on g-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 2172
   - name: g_gaap0p5Flux
     "@id": "#Object.g_gaap0p5Flux"
     datatype: double
     description: GaaP flux with 0.5 aperture after multiplying the seeing by 1.15. Forced on g-band.
     fits:tunit: nJy
+    tap:principal: 0
```

Comment:
I don't know what the phrase "multiplying the seeing" means.  @arunkannawadi, do you?

Mention of aperture size needs units, too.

Reply:
The description makes it sound like a completely arbitrary kludge.